apps/studio/package.json

```diff
@@ -11,7 +11,7 @@
     "db:reset": "prisma migrate reset",
     "db:push": "prisma db push",
     "setup": "run-s --print-label db:setup db:sleep migrate:dev db:seed",
-    "setup:test": "run-s --print-label db:setup db:sleep migrate generate db:seed",
+    "setup:test": "run-s --print-label db:setup db:sleep migrate generate",
```

Comment:
omitting seed implies that tests have to scaffold the data themselves. i thought the `seed` script provided a sane base for tests to use and if they have any exceptional cases, they'd scaffold per test.

do you foresee us reimplementing the seed script in tests?

Reply:
this one got circular dependency. maybe we try to fix in the future. cause db:seed uses kysely db instance, which uses isomer-components. but when it uses, the components not built yet. Then fail lol. Tackle in future

Reply:
curerently the scuffed way is to commit the built isomer-components.tgz into the repo, but that is very weird and should not be needed.